sklearn/naive_bayes.py

```diff
@@ -212,6 +212,8 @@ class labels known to the classifier.
     [1]
     """
 
+    _parameter_constraints = {"priors": ["array-like", None], "var_smoothing": [Real]}
```

Comment:
var_smoothing must be positive
```suggestion
    _parameter_constraints = {
        "priors": ["array-like", None],
        "var_smoothing": [Interval(Real, 0, None, closed="left")],
    }
```

Reply:
Updated the constraint for `var_smoothing` parameter to be positive real number.